app/src/main/java/com/codesoom/assignment/utils/JwtUtil.java

```diff
@@ -0,0 +1,44 @@
+package com.codesoom.assignment.utils;
+
+import com.codesoom.assignment.errors.InvalidTokenException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+
+@Component
+public class JwtUtil {
+    private final Key key;
+
+    public JwtUtil(@Value("${jwt.secret}") String secret) {
+        this.key = Keys.hmacShaKeyFor(secret.getBytes(StandardCharsets.UTF_8));
+    }
+
+    public String encode(Long userId) {
+        return Jwts.builder()
+                .claim("userId", userId)
+                .signWith(key)
+                .compact();
+    }
+
+    public Claims decode(String token) {
+        if (token == null || token.isBlank()) {
```

Comment:
```suggestion
        if (Strings.isBlank(token)) {
```
이런식으로 바꾼다면 좀더 가독성이 올라갈 것 같아요!

Reply:
오 감사합니다!